tests/system/action/user/test_create.py

```diff
@@ -361,123 +585,95 @@ def test_create_permission_manage_user(self) -> None:
-                "is_physical_person": True,
-                "default_password": "new default_password",
-                "gender": "female",
-                "email": "info@openslides.com ",
-                "default_number": "new default_number",
-                "default_structure_level": "new default_structure_level",
-                "default_vote_weight": "1.234000",
-                "organisation_management_level": OrganisationManagementLevel.CAN_MANAGE_USERS,
                 "committee_ids": [60, 63],
+                "committee_$60_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "committee_$63_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "username": "usersname",
             },
         )
 
-    # TODO: fix when committee permission system is implemented
-    @pytest.mark.skip()
-    def test_create_permission_user_can_manage(self) -> None:
-        """ May create all group-fields """
+    def test_create_permission_group_D_permission_with_CML(self) -> None:
+        """
+        May create Group D committee fields with CML permission.
+        The new user can be manager of more than one Committees, but only member of 1 committee.
+        For being member in 2 committees the request user has to have OML permissions for organisation scope.
+        """
         self.permission_setup()
         self.create_meeting(base=4)
-        self.set_organisation_management_level(
-            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        self.update_model(
+            f"user/{self.user_id}",
+            {
+                "committee_$60_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "committee_$63_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "committee_$_management_level": ["60", "63"],
+            },
         )
-        self.set_user_groups(
-            self.user_id, [2, 6]
-        )  # admin-group of meeting/1 and group of meeting/4
-        self.set_group_permissions(6, [Permissions.User.CAN_MANAGE])
-        self.create_user("user3", [4])
-        self.create_user("user4", [4])
 
         response = self.request(
             "user.create",
             {
-                "username": "username",
-                "number_$": {"1": "number1", "4": "number1 in 4"},
-                "structure_level_$": {"1": "structure_level 1"},
-                "vote_weight_$": {"1": "12.002345"},
-                "about_me_$": {"1": "about me 1"},
-                "comment_$": {"1": "comment zu meeting/1"},
-                "vote_delegated_$_to_id": {"1": self.user_id},
-                "vote_delegations_$_from_ids": {"4": [3, 4]},
-                "group_$_ids": {"1": [2, 3], "4": [5]},
+                "username": "usersname",
+                "committee_$_management_level": {
+                    "60": CommitteeManagementLevel.CAN_MANAGE,
+                    "63": CommitteeManagementLevel.CAN_MANAGE,
```

Comment:
Shouldn't it be forbidden to set invalid replacements here?

Reply:
They are valid. self.create_meeting creates a committee with meeting + offset 59.